src/main/java/signup_and_login/UserOutputBoundary.java

```diff
@@ -0,0 +1,6 @@
+package signup_and_login;
+
+public interface UserOutputBoundary {
+    void prepareJoinOrHostView(String userID);
```

Comment:
userID should be an attribute of user that is uniquely generated each time a new user is created

Reply:
see commit 447b882605d4d15fc8cb8bd701de977f6cbfa13e